peerid.go

```diff
@@ -7,12 +7,14 @@ import (
 	"github.com/ethereum/go-ethereum/crypto"
 )
 
-type PeerId ecdsa.PublicKey
+type PeerId [64]byte
```

Comment:
Usually in Go, the naming convention for shortcuts and keywords is to uppercase them:
```suggestion
type PeerID [64]byte
```

similarly with `HTTPStatus` or `MarshalJSON` etc.